ember-moment/babel.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",
+    ["babel-plugin-ember-template-compilation", {
+      "targetFormat": "hbs",
+      "transforms": []
+    }],
+    ["module:decorator-transforms", { "runtime": { "import": "decorator-transforms/runtime" } }],
```

Comment:
Does ember-moment use decorators?

If so, using this plugin has requirements of the consuming project (ember-cli-babel 8.2+)

Reply:
looked into code... no there are not decorator... but i have discovered seen, that we have still `classic classes` in this addon 😮 (see service)

Reply:
`decorator-transforms` is now removed. thank you for hint

Reply:
Excellent, thank you!